lib/datadog/tracing/contrib/rails/configuration/settings.rb

```diff
@@ -49,6 +49,8 @@ def initialize(options = {})
             end
 
             option :distributed_tracing, default: true, type: :bool
+            option :request_queuing, default: false, type: :bool
```

Comment:
this line is probably not needed here as this option is still defined on line 55

Reply:
Removed at cb9ccbe